### PR TITLE
IR-docs-v4

### PR DESCRIPTION
**Pull request by justinegeffen**
Just some content updates.

One idea I have for the next version is using videos/use cases in some of the sections. Do we have any marketing collateral around use cases/workflows/post-mortems? It would be ideal to link out to them as these are purely technical docs but links to demos etc would enhance the content.

@itao, thoughts on that?

### CODE REVIEW

File: docs/playbooks/playbook-overview.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
...
+# Playbooks and Incidents
 
 During an active incident, you want to focus on triaging and solving the problem as soon as possible. Planning your incident support strategy ahead of time with playbooks is the best way to ensure incidents run smoothly.
 
-## What are playbooks?
-
-A playbook is a recipe for dealing with and resolving an incident. In a playbook, you can plan ahead so that during an incident responders know exactly what to do. Make sure to schedule a retrospective analysis to iterate on the design of your playbooks after the incident finishes.
-
-Within each playbook, you can create stages and tasks to ensure that items are addressed and completed in sequential order. The tasks can optionally be associated with slash commands and assigned to individual team members.
+Once complete, incident channels can be exported for analysis. Teams can identify bottlenecks in the incident by seeing time gaps between when checklist items are completed and incorporating necessary changes into the playbook for next incident.
```
Comment (lieut-data) [inline docs/playbooks/playbook-overview.md:5]: Should we call out the channel export plugin here, or is the previous link sufficient?

**Review by lieut-data — APPROVED** [review]

File: docs/incidents/ending-an-incident.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
-Ending an incident signals to all participants that the issue has been resolved. Some playbooks may define stages and tasks to complete after an incident has been resolved, such as scheduling and completing a post-mortem. The incident RHS remains fully accessible even after an incident has ended.
+![Ending an incident](../assets/incident_end_slash_command.png)
+
+![Confirmation](../assets/confirm_end_incident.png)
+
+An ended incident can be restarted at any time using `/incident restart` from within the incident channel or the **Incident Restart** in the RHS.
+
+![Restart an incident](../assets/incident_restart_slash_command.png )
+
+Some playbooks may define stages and tasks to complete after an incident has been resolved, such as scheduling and completing a post-mortem.
```
Comment (justinegeffen) [inline docs/incidents/ending-an-incident.md:13]: @lieut-data, confirming that this is still accurate?

File: docs/playbooks/playbook-overview.md (hunk 1 above)
Comment (justinegeffen) [inline docs/playbooks/playbook-overview.md:5]:
```suggestion
Once complete, [incident channels can be exported](https://mattermost.gitbook.io/channel-export-plugin) for analysis. Teams can identify bottlenecks in the incident by seeing time gaps between when checklist items are completed and incorporating necessary changes into the playbook for next incident.
```

File: docs/incidents/ending-an-incident.md (hunk 2 above)
Comment (lieut-data) [reply docs/incidents/ending-an-incident.md:13]: Yes, this remains accurate.

**Comment by justinegeffen** [comment]
@lieut-data, this is good to go for this iteration.

File: docs/incidents/ending-an-incident.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
 
-Incident members can end an incident via either a slash command or the right-hand sidebar.
+Incident members can end an incident using the slash command `/incident end` from within the incident channel. Ending an incident signals to all participants that the issue has been resolved.
 
-Ending an incident signals to all participants that the issue has been resolved. Some playbooks may define stages and tasks to complete after an incident has been resolved, such as scheduling and completing a post-mortem. The incident RHS remains fully accessible even after an incident has ended.
+![Ending an incident](../assets/incident_end_slash_command.png)
+
+![Confirmation](../assets/confirm_end_incident.png)
+
+An ended incident can be restarted at any time using `/incident restart` from within the incident channel or the **Incident Restart** in the RHS.
```
Comment (agarciamontoro) [inline docs/incidents/ending-an-incident.md:9]:
```suggestion
An ended incident can be restarted at any time using `/incident restart` from within the incident channel or with the **Restarr Incident** button in the RHS.
```

File: docs/playbooks/creating-and-managing-playbooks.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+4. Name your playbook.
+5. Edit the **Default Stage**, defining one or more steps to be taken by members of the incident.
+   * Optionally use descriptions on steps to add additional context for members of the incident. Descriptions support a limited form of markdown, including text styling and hyperlinks.
+   * Optionally define a slash command with the step, simplifying the completion of steps in the incident.
+6. Configure whether the incident channel should be public or private within the team.
+7. Share this playbook with other members of the team to allow them to use the playbook to start an incident, as well as edit the contents.
+
+## Editing a playbook
+
+You can edit a playbook that's being used in an active incident. However, the changes will only be applied to future incidents - not the active incident, or incidents that previously used that playbook.
```
Comment (agarciamontoro) [inline docs/playbooks/creating-and-managing-playbooks.md:22]: I feel like we can remove the "that's being used in an active incident". Something like 
```suggestion
You can edit a playbook at any time. However, the changes will only be applied to future incidents - not the active incidents, or incidents that previously used that playbook.
```

**Review by agarciamontoro — CHANGES_REQUESTED** [review]
Great! I left a couple of comments below. I also took a look at the screenshots, and `playbook_stages_edit.png`, `incident_response.png` and `incident_response_center.png` are kind of out of focus (it looks like an issue when changing their size or converting to png or something similar)

File: docs/incidents/ending-an-incident.md (hunk 3 above)
Comment (agarciamontoro) [reply docs/incidents/ending-an-incident.md]: :point_up: I'm not sure if we need the "with" and the "button", but the button does read "Restart Incident", not "Incident Restart".

File: README.md [hunk 5]
```diff
@@ -6,7 +6,7 @@
 
 Incident Response allows your team to coordinate, manage, and resolve incidents from within Mattermost. For configuration and administration information visit our [documentation](https://mattermost.gitbook.io/incident-response-plugin/).
 
-![Incident Response plugin screenshot](assets/center_channel.png)
+![Incident Response plugin screenshot](assets/incident_response.png)
```
Comment (justinegeffen) [inline README.md:9]:
```suggestion
![Incident Response plugin screenshot](assets/incident_screen.png)
```

**Comment by justinegeffen** [comment]
Thanks, @agarciamontoro! I've fixed one of the screenshots but the incident response main screenshot is still not perfect. Could we merge this so I can see it in prod before I decide whether to remove for now?

**Review by agarciamontoro — APPROVED** [review]
Sounds good!

**Comment by justinegeffen** [comment]
@agarciamontoro, you'll need to merge for me. :D

**Comment by agarciamontoro** [comment]
There was a failing E2E test here, probably a flaky one: https://mattermost.atlassian.net/browse/MM-29745

**Pull request merged**